terraform/pastebin.tf

```diff
@@ -0,0 +1,47 @@
+module "pastebin-stage-db" {
+  source = "git::https://github.com/mozilla-it/terraform-modules.git//aws/database"
+
+  identifier = "pastebin-${var.environment}"
+  name       = "pastebin"
+  storage_gb = 20
+  subnets    = data.terraform_remote_state.vpc.outputs.private_subnets
+  type       = "mysql"
+  username   = "pastebin"
+  vpc_id     = data.terraform_remote_state.vpc.outputs.vpc_id
+
+  cost_center = var.cost_center
+  environment = var.environment
+  project     = "pastebin"
+}
+
+# Pastebin Application Secrets Metadata
+
+resource "aws_secretsmanager_secret" "db_host" {
+  name        = "websre/pastebin/env/database_host"
+  description = "pastebin stage DB_HOST"
+}
+
+resource "aws_secretsmanager_secret" "db_name" {
+  name        = "websre/pastebin/env/database_name"
```

Comment:
do we expect the literal env here?  I was expecting that to be 'prod' or 'stage' or what have you.

Reply:
shit yeah, i caught that back in pastebin-infra. the idea is env would be replaced by ${var.environment} (or whatever setup to get the current environment interpolated there).

Reply:
going to fix in option 1 branch, thanks for catching